pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,132 @@
+"""DP computing of count, sum, mean, variance."""
+
```

Comment:
Please rename a file to dp_computations.py
You can do  `git mv <old name> <new name>` it should save all comments